src/app/containers/Ad/Amp/index.test.jsx

```diff
@@ -50,16 +59,26 @@ describe('AMP Ads', () => {
   });
 
   describe('Assertions', () => {
-    it('should not render ad placeholder in UK', () => {
+    it('should not render ad placeholder in UK when adPlaceholder toggle is false', () => {
       const { getByLabelText } = render(
         <div className="amp-geo-group-eea amp-geo-group-gbOrUnknown">
-          {adWithContext('leaderboard')}
+          {adWithContext('leaderboard', false)}
         </div>,
       );
 
       expect(getByLabelText('Publicités')).not.toBeVisible();
     });
 
+    it('should render ad placeholder in UK when adPlaceholder toggle is true', () => {
```

Comment:
I think there maybe a few missing tests here around making sure the placeholder doesn't show when ads are disabled?

Reply:
We check if ads are enabled [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Ad/index.jsx#L13) so the placeholder will not be shown

Reply:
I was more thinking about test coverage here; the tests should assume we know nothing about how the code works. If someone changes the structure of the component later we should still have test coverage to ensure the ad placeholder doesn't show when ads are disabled imo.

Reply:
Added tests for when adsEnabled is false in the [advert component](https://github.com/bbc/simorgh/pull/9209/files#diff-362bc4c4b9894c2f471cc42d9cf6b053ca3dacb38c7a3e53aadd0d7ec5021954R192), setting placeholder to true for [AMP](https://github.com/bbc/simorgh/pull/9209/files#diff-362bc4c4b9894c2f471cc42d9cf6b053ca3dacb38c7a3e53aadd0d7ec5021954R211)